src/app/ApplicationTemplate.Tests/Business/UserProfileServiceShould.Mock.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ApplicationTemplate.Client;
+
+namespace ApplicationTemplate.Tests.Business;
+
+public sealed partial class UserProfileServiceShould
```

Comment:
Is it really worth putting in a different file?

Reply:
@jeanplevesque I initially intended to create this mock data file to test the UpdateProfile method in the UserProfileService, but I changed my strategy so now, it only has 1 mock profile.